src/sections/OurDoctors/DoctorSectionWrapper.jsx

```diff
@@ -0,0 +1,17 @@
+import Paragraph from "../../components/Paragraph";
+import HeaderTitle from "./HeaderTitle";
+import TeamSlider from "./TeamSlider";
+
+const DoctorSectionWrapper = () => {
```

Comment:
Remove `OurDoctors` folder and keep this directly in sections.

Also renamed to `TopDoctorsSection`.

Reply:
Solved it. thank you!